src/Microsoft.HttpRepl/Commands/UICommand.cs

```diff
@@ -17,6 +16,12 @@ namespace Microsoft.HttpRepl.Commands
     public class UICommand : ICommand<HttpState, ICoreParseResult>
     {
         private static readonly string Name = "ui";
+        private IUriLauncher UriLauncher;
+
+        public UICommand(IUriLauncher uriLauncher = null)
+        {
+            UriLauncher = uriLauncher ?? new UriLauncher();
```

Comment:
Don't allow null in the constructor.  Force the dependency to be provided.

Reply:
Updated here:
https://github.com/aspnet/HttpRepl/commit/dc0c5d83e599b9d8462fd595ae2e8e8e35d527cc